onnx/__init__.py

```diff
@@ -62,12 +62,15 @@ def _serialize(proto: Union[bytes, google.protobuf.message.Message]) -> bytes:
     '''
     if isinstance(proto, bytes):
         return proto
-    elif hasattr(proto, 'SerializeToString') and callable(proto.SerializeToString):
-        result = proto.SerializeToString()
+    if hasattr(proto, 'SerializeToString') and callable(proto.SerializeToString):
+        try:
+            result = proto.SerializeToString()
+        except:
```

Comment:
I would catch a more specific exception

Reply:
All updated. Thanks for the reviews. Please review it again.